client/src/Pos/Client/Txp/Util.hs

```diff
@@ -65,7 +65,7 @@ import           Pos.Core (Address, Coin, StakeholderId, TxFeePolicy (..), TxSiz
                            bvdTxFeePolicy, calculateTxSizeLinear, coinToInteger, integerToCoin,
                            isRedeemAddress, mkCoin, protocolMagic, sumCoins, txSizeLinearMinValue,
                            unsafeIntegerToCoin, unsafeSubCoin)
-import           Pos.Core.Configuration (HasConfiguration)
+import           Pos.Core.Configuration (HasConfiguration, HasProtocolMagic)
```

Comment:
I cleaned up some of the excess constraints to make it easier to reuse these functions. Not necessary to roll in.